src/tests/functional/inference_engine/transformations/smart_reshape/lstm_states_broadcast.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <openvino/core/model.hpp>
+#include <openvino/opsets/opset9.hpp>
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+
+template<class T>
+std::shared_ptr<ov::opset9::Constant> create_constant(const std::vector<T>& data, const ov::element::Type_t et = ov::element::i64, bool scalar = false) {
```

Comment:
This is same in this tests and also in reshape_sinking tests, can move as some test util function?